tests/conftest.py

```diff
@@ -598,6 +598,14 @@ def collection(db):
     return collection
 
 
+@pytest.fixture
+def not_yet_available_collection(db):
```

Comment:
Once again, saying:
> is collection available

Is really odd for a collection (plus the test `test_collection_not_yet_published` is having a different name).